_docs/welcome/notes/0.4.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Istio 0.4
+
+order: 97
+
+layout: docs
+type: markdown
+---
+
+## General
+
+This release has only got a few weeks worth of changes, as we stabilize our monthly release process. 
+In addition to the usual pile of bug fixes and perf improvements, this release includes:
+
+- **Cloud Foundry**. Added support for the [Cloud Foundry](https://www.cloudfoundry.org) platform to Pilot, making it
+possible for Pilot to discover CF services and service instances. This represents the strict minimum needed to
+get things running, additional work remains in this area to flesh things out.
+
+- **Circonus**. Mixer now includes an adapter for the [Circonus](https://www.circonus.com) analytics and monitoring platform.
```

Comment:
Is our plan to track integrations in our release notes?  Is this something that should be tracked and announced in a different fashion?

Reply:
For now, these adapters are built in to Mixer so I think it's worth tracking here. Once we establish the extensions repo and move adapters there, then we probably won't announce adapters as part of the main Istio release notes.